lib/licensee/project_files/license_file.rb

```diff
@@ -9,6 +9,10 @@ class LicenseFile < Licensee::ProjectFiles::ProjectFile
       PREFERRED_EXT = %w[md markdown txt html].freeze
       PREFERRED_EXT_REGEX = /\.#{Regexp.union(PREFERRED_EXT)}\z/.freeze
 
+      # List of short license names that can be used as extensions
+      LICENSE_NAME_EXT = %w[AGPLv3 GPLv3 LGPLv3 MIT MPLv2].freeze
```

Comment:
These specific extensions should be in a test rather than added here.

I'm not sure why eg LICENSE.AVPLv3 isn't being matched by line 45 below but would suggest LICENSE.* should be matched barring a few known non-license texts (spdx, header, xml, etc below).

Reply:
Ah, it does match per https://github.com/licensee/licensee/issues/638#issuecomment-1444212997 so closing this PR.